Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -3233,9 +3235,9 @@ class BlurWrapper: UIView {
             }
             effectView.hidden = disableBlur
             switch blurStyle {
-            case .ExtraLight, .Light:
+            case .ExtraLight, .Light, .Regular, .Prominent:
```

Comment:
From a transcipt of a WWDC talk [1]

> So if you use UIBlurEffectStyleRegular and the system's in light, it will use UIBlurEffectStyle.light.
> If you use regular and dark, you'll use dark.
> If you use prominent, it will use .extraLight and .extraDark. 

I think for our uses in this method we want to ignore the system blur and just stick with what we were doing.

[1] http://asciiwwdc.com/2016/sessions/206